config.py

```diff
@@ -1,36 +1,67 @@
+"""SelfDrive Environement config
+
+Use this file to help customize various aspects of the simulations. If you find
+additional things in the simulation that you'd like to customize (and think
+others probably would as well) feel free to add it in and open a pull request on
+the repo: https://github.com/Taaseen-Ali/OpenAI-Gym-Car-Race
+"""
+                                
+# Default car settings
+
 car = {
-    "position" : [300, 100],
-    "angle" : 90,
-    "num_sensors" : 10,
-    "speed" : 0,
-    "rotation" : 0
+    # Initial defaults
+
+    "position": [100, 100],     # Starting x, y position of the car. Note that
+                                # this will be deprecated soon because of
+                                # https://github.com/Taaseen-Ali/OpenAI-Gym-Car-Race/pull/7
```

Comment:
lol noice, dat be done and already incorporated within your commit

Reply:
yep!. I'll open a baby issue for someone else on the team to get rid of the stuff we don't need any more thanks to the work you did with start/finish. Ease them into doing bigger tasks.